src/content/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration.mdx

```diff
@@ -34,18 +34,29 @@ Go to the [Prometheus remote write setup launcher in New Relic One](https://one.
 
    Use the following syntax:
 
+   **Prometheus v2.26 and newer**
+   ``` 
+   remote_write: 
+   - url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=<var>YOUR_DATA_SOURCE_NAME</var>  
+     authorization:
+     credentials: YOUR_LICENSE_KEY
```

Comment:
I *think* `credentials` needs an extra indentation to put it inside the `authorization` map? or maybe YAML will just do the right thing.

Reply:
Thanks, @haus!